### DISCUSSION TITLE
[Support] QA：使用@midwayjs/sequelize如何使用事务？

### DISCUSSION THREAD

**Discussion by zhhbinn** [midwayjs/midway#3550] [Q&A] [answered]
### Describe the problem（描述问题）

使用@midwayjs/sequelize如何开启事务？最好是自动托管事务。看到typeorm有详细的相关文档，但是sequelize的文档中没有提及事务相关

### Midway Versions（Midway 版本）

✓ @midwayjs/sequelize(3.13.9)

**Comment by czy88840616** [comment]
sequelize-typescript 看起来并没有提供事务处理的装饰器等方法，目前仅能通过底层 sequelize 的 api 去处理。 https://sequelize.org/docs/v6/other-topics/transactions/

**Reply by zhhbinn** [reply]
请问如何拿到Sequelize的构建器，以做到“自动将事务传递给所有查询”？
即文档中的这种方式：https://sequelize.org/docs/v6/other-topics/transactions/#automatically-pass-transactions-to-all-queries

**Reply by czy88840616** [reply, marked as answer]
就按他写的，require('sequelize'); 就行
